src/modules/storage/LedgerStorage.ts

```diff
@@ -235,4 +348,403 @@ export class LedgerStorage extends Storages
             callback(err, rows);
         });
     }
+
+    /**
+     * Get enrollments
```

Comment:
Invalid getValidator function and description.

Reply:
I fixed it. thanks